.changeset/twenty-yaks-sell.md

```diff
@@ -0,0 +1,6 @@
+---
+"tinacms": patch
```

Comment:
From my understanding, @tinacms/cli should be included in the changeset.

```suggestion
"tinacms": patch
"@tinacms/cli": minor
```

Reply:
I've discarded the import change so the changeset change isn't required